tests/test_strds_raster_renderer.py

```diff
@@ -34,31 +34,27 @@
 
 
 __license__ = "GPLv3"
-__author__ = "Sören Gebbert"
+__author__ = "Sören Gebbert, Anika Weinmann"
 __copyright__ = (
-    "Copyright 2016-2018, Sören Gebbert and mundialis GmbH & Co. KG"
+    "Copyright 2016-2024, Sören Gebbert and mundialis GmbH & Co. KG"
 )
 __maintainer__ = "Soeren Gebbert"
 __email__ = "soerengebbert@googlemail.com"
 
-location = "nc_spm_08"
+project = "nc_spm_08"
 strds_mapset = "modis_lst"
-strds_url = URL_PREFIX + "/locations/%(location)s/mapsets/%(mapset)s/strds" % {
-    "location": location,
-    "mapset": strds_mapset,
-}
+# strds_url = (
+#     f"{URL_PREFIX}/PROJECT_URL_PART/{project}/mapsets/{strds_mapset}/strds"
+# )
```

Comment:
Why is this outcommented? Is this not needed?

Reply:
Yes it is not used